### DISCUSSION TITLE
https://cucoders.dev/publicaciones/carlosjorger/como-crear-una-librer-a-para-vue3/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#126] [Page Comments]
# https://cucoders.dev/publicaciones/carlosjorger/como-crear-una-librer-a-para-vue3/

En este artículo explicare paso a paso como crear un librería para Vue3 usando Vite** y **Typescript.

https://cucoders.dev/publicaciones/carlosjorger/como-crear-una-librer-a-para-vue3/

**Comment by manuelernestog** [comment]
Que bueno el artículo bro 👏👏. Super

**Reply by carlosjorger** [reply]
Muchas gracias hermano🙌

**Reply by manuelernestog** [reply]
🙌🙌

**Comment by JimScope** [comment]
Me gustó. ¿Podrías dar algunos detalles de que debe llevar un componente típico (Ejemplo. Un Alert)?

**Reply by carlosjorger** [reply]
te refieres a la estructura de la componennte de vue?
